helm/prometheus-rules/templates/alerting-rules/aws.management-cluster.rules.yml

```diff
@@ -146,4 +146,16 @@ spec:
         severity: page
         team: phoenix
         topic: kubernetes
+    - alert: IRSATooManyErros
+      annotations:
+        description: '{{`IRSA bootstrap for Cluster {{ $labels.cluster_id }} failed.`}}'
+        opsrecipe: irsa-operator-error/
+      expr: irsa_operator_cluster_errors > 0
```

Comment:
0 seems to me a very low threshold.
Also, would it be possible to make it a SLO-based alert?

Reply:
The thing is whenever there's one error in the chain there is something wrong and it will not recover. 

My intention was to make it really noisy in the beginning so we can probably smash all errors in case there are some or we immediately fix for instance the permissions if they are not set right and later we turn it into something SLO like because we have more experience and it will unlikely that there are bigger issues

Reply:
Fine with me when that's a conscious choice.

Reply:
okay